app/src/main/java/com/example/ar_reshare/MessageListAdapter.java

```diff
@@ -22,12 +25,16 @@ public class MessageListAdapter extends RecyclerView.Adapter {
 
     private Context mContext;
     private List<Message> mMessageList;
+    private Message.MessageResult messageResult;
+    int loggedInUserID;
```

Comment:
Could you please add a _public_ access modifier here? Sorry, this is my fault I should have done it in my last pull request!

Reply:
Actually, my mistake this is the wrong file. Feel free to leave it with the default or change it to private. (resolved)